featurehub-javascript-client-sdk/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+#### 1.1.8
```

Comment:
We will have to do a breaking major version increase due to the default polling change. Also it doesn't seem to be in the notes